WebContent/WEB-INF/jsp/dataSourceEdit/editMqtt.jsp

```diff
@@ -60,6 +66,13 @@
     DataSourceEditDwr.saveMqttPointLocator(
     currentPoint.id, $get("xid"), $get("name"), locator, savePointCB);
     }
+  function isValid(value) {
+    return value == "" || isPositiveInt(value);
+    }
+
+  function isPositiveInt(value) {
```

Comment:
Identical functions (isValid, isPositiveInt ) are in events.jsp, so they need to be moved to common.js